web/template/partial/stream/chat/chat_message.gohtml

```diff
@@ -27,7 +27,16 @@
                     lg:hover:bg-gray-100 lg:dark:hover:bg-gray-700
                 {{end}}">
                 <div class="flex justify-between">
-                    <span class="chatMsg w-5/6 overflow-wrap-anywhere my-auto" x-html="m.message"></span>
+                    <span class="chatMsg w-5/6 overflow-wrap-anywhere my-auto" x-init="$nextTick(() => { renderMathInElement($el, {
```

Comment:
Instead of doing everything inline in the html file you could utilize the preprocessors of the `Chat` object like so:

```typescript 
import katex from "katex";
...
 preprocessors: ((m: ChatMessage) => ChatMessage)[] = [
        ...
        (m: ChatMessage) => {
            const msg = removeDelimiters(m.message); 
            m.message = katex.renderToString(msg, {throwOnError: false});
            return m;
        },
        ...
    ];
...
```
Then the auto-render dependency isn't necessary anymore. 

However, there will be no delimiters anymore. For example instead of `$\frac{1}{2}$`,  `\frac{1}{2}`  will suffice.

To make copy-tex work again a `removeDelimiters` function which extracts the content in between two delimiter symbols (e.g. `$<math>$`) would be needed.